doc/source/release-notes/Percona-Server-5.7.21-20.rst

```diff
@@ -0,0 +1,122 @@
+.. rn:: 5.7.21-20
+
+========================
+Percona Server 5.7.21-20
+========================
+
+Percona is glad to announce the release of Percona Server 5.7.21-20
+on February 16, 2018. Downloads are available `here
+<http://www.percona.com/downloads/Percona-Server-5.7/Percona-Server-5.7.21-20/>`_
+and from the :doc:`Percona Software Repositories </installation>`.
+
+This release is based on `MySQL 5.7.21
+<http://dev.mysql.com/doc/relnotes/mysql/5.7/en/news-5-7-21.html>`_
+and includes all the bug fixes in it.
+Percona Server 5.7.21-20 is now the current GA (Generally Available) release
+in the 5.7 series. All software developed by Percona is open-source and free.
+
+New Features
+============
+
+* A new string variable :variable:`version_suffix` allows to change suffix
+  for the |Percona Server| version string returned by the read-only
+  :variable:`version` variable. Also :variable:`version_comment` is converted
+  from a global read-only to a global read-write variable.
+
+* A new :variable:`keyring_vault_timeout` variable allows to set the amount
+  of seconds for the Vault server connection timeout. Bug fixed :psbug:`298`.
+
+Bugs Fixed
+==========
+
+* mysqld startup script was unable to detect jemalloc library location for
+  preloading, and that prevented starting |Percona Server| on systemd based
+  machines. Bugs fixed :psbug:`3784` and :psbug:`3791`.
+
+* There was a problem with fulltext search, which could find a word with
+  punctuation marks in natural language mode only, but not in boolean mode.
+  Bugs fixed :psbug:`258`, :psbug:`2501` (upstream :mysqlbug:`86164`).
+
+* Build errors were present on FreeBSD (caused by fixing the bug
+  :psbug:`255` in |Percona Server| :rn:`5.6.38-83.0`) and on MacOS (caused
+  by fixing the bug :psbug:`264` in |Percona Server| :rn:`5.7.20-19`). Bugs
+  fixed :psbug:`2284` and :psbug:`2286`.
+
+* A bunch of fixes where introduced to make |Percona Server| compile under
+  GCC 7. Bugs fixed :psbug:`3780` (upstream :mysqlbug:`89420`,
```

Comment:
warning fixes (not errors, thus it was possible to compile under GCC 7 already, if needed with warnings-as-errors relaxed)